### ISSUE TITLE
Question: Where Do TS* Commands Come From?

### ISSUE THREAD

**Issue by atomdmac** [neovim/neovim#13328]
NVIM Version: 0.4.4

This is not a bug report (and may be the wrong place entirely for this question) but I'm at a loss for a better place to start my investigation:

I noticed recently that a number of `TS*` commands are available in `neovim`.  (Ex. `TSDef`, `TSType`, `TSEditConfig`, etc.).  I've removed all plugins (i.e. cleared out my `init.vim` and done a `:PlugClean`) but the commands are still available in the auto-complete in NVIM.

The commands don't appear to do anything and produce no output (even when my current working directory is a TypeScript project).

Does anyone know where these commands are coming from or where I can find out more about how they're supposed to work?

**Comment by vigoux** [comment]
Did you tried `nvim - u NONE` and see if those commands are still there?
I can't reproduce on my side...

**Comment by bfredl** [comment]
Sure you don't have Nvim 0.5 files laying around? Tree sitter is a 0.5 feature.

**Comment by clason** [comment]
These look like they come from https://github.com/mhartington/nvim-typescript, so it seems that you are not actually testing your minimal config?

**Comment by atomdmac** [comment]
This definitely seems to be an issue where some plugins/scripts were still getting auto-loaded somehow (though I'm not sure from where).  I recently reformatted my Mac and did _not_ have `nvim-typescript` installed.  That said, running `nvim -u NONE` caused the commands to disappear.

As this sounds like something related to my specific setup, I'm going to close this issue.  Thanks to everyone for chiming in!

P.S.  After seeing @clason 's comment of the possible origin being `nvim-typescript`, I installed it and tried the commands again.  After doing do, the commands _do_ perform as expected (instead of having no effect or output).
